tools/osx/crosstool/cc_toolchain_config.bzl

```diff
@@ -5194,20 +5194,12 @@ def _impl(ctx):
     else:
         unfiltered_compile_flags_feature = None
 
+    do_not_split_linking_cmdline = feature(name = "do_not_split_linking_cmdline")
```

Comment:
https://github.com/bazelbuild/bazel/issues/7687

Reply:
I'm confused, why do you need to add the feature? That flag was flipped

Reply:
I found without this this didn't work at all and no arguments were passed to the linker.

Reply:
The reason for that must be this: https://cs.opensource.google/bazel/bazel/+/master:src/main/java/com/google/devtools/build/lib/rules/cpp/CppActionConfigs.java;drc=a5a4f47295ccb6cd81cf8732772c849b1a8b9d58;l=1500

Can you make sure you don't have the flag `incompatible_do_not_split_linking_cmdline` anywhere? Bazelrc perhaps?

This feature and the flag should eventually go away.

Reply:
I definitely don't have that one set. I can try it in a more isolated environment though.

Reply:
My assumption is when that flag went away the checks that required this would also go away and we could delete this then

Reply:
ok I'm not sure what was happening in my case before, but you're right this isn't needed.